### PR TITLE
General cleanup

### PR DESCRIPTION
**Pull request by kritzcreek**

### CODE REVIEW

File: internal/document/document.go [hunk 1]
```diff
@@ -93,7 +93,7 @@ func (d *Document) SetNewSymbolForPos(
 			documentation = append(documentation, symbols.FormatCode(signature))
 		}
 		if hover != "" {
-			documentation = append(documentation, symbols.FormatMarkdown(hover))
+			documentation = append(documentation, hover)
```
Comment (keynmol) [inline internal/document/document.go:96]: So in this case is `hover` the one that will contain Go Doc comment?

What is `extra` then? And I'm guessing `signature` is a true symbol signature, without any docs?

**Review by keynmol — APPROVED** [review]
I have truly mixed feelings about this :D 
image

File: internal/document/document.go (hunk 1 above)
Comment (kritzcreek) [reply internal/document/document.go:96]: I don't know, but the function this was calling was:
```
func FormatMarkdown(v string) string {
	if v == "" {
		return ""
	}

	// var buf bytes.Buffer
	// doc.ToMarkdown(&buf, v, nil)
	// return buf.String()
	return v
}
```
which is just identity for strings, so I removed both the function and its one call-site.

Comment (keynmol) [reply internal/document/document.go:96]: ahaha. Okay.

**Pull request merged**